### PR TITLE
pagination-updated-mayorisaac

### PR DESCRIPTION
**Pull request by Mayorisaac24**
I update the pagination. You can disregard if any other changes aside that:     
if dataset != None:
        start = (page - 1) * limit
        end = start + limit
        return dataset[start:end]
return None

### CODE REVIEW

File: app/helpers.py [hunk 1]
```diff
@@ -42,9 +42,11 @@ def get_query_param_value_from_url(url, query_param):
...
 def paginate(dataset, page, limit):
     """Paginate/Chunk Results"""
-    start = (page - 1) * limit
-    end = start + limit
-    return dataset[start:end]
+    if dataset != None:
+        start = (page - 1) * limit
+        end = start + limit
+        return dataset[start:end]
+    return None
```
Comment (WawinyEdwin) [inline app/helpers.py:49]: Return an Empty list instead.

File: app/views.py [hunk 2]
```diff
@@ -284,8 +284,7 @@ def finalize_or_reject(request, process_id):
             background = Background(process, item_type, item_id, role, user)
             background.processing()
             if user_type == "public":
```
Comment (WawinyEdwin) [inline app/views.py:286]: Why do we have this change?

**Review by WawinyEdwin — CHANGES_REQUESTED** [review]
Make the changes so we can merge this

File: app/helpers.py (hunk 1 above)
Comment (Mayorisaac24) [reply app/helpers.py:49]: Okay

File: app/views.py (hunk 2 above)
Comment (Mayorisaac24) [reply app/views.py:286]: I didn't make that change. I made a pull request maybe it didn't update completely before I make the changes. I will make a pull request again and see

**Comment by Mayorisaac24** [comment]
It has been resolved and pagination was added to some other functions

**Review by WawinyEdwin — APPROVED** [review]

**Pull request merged**